src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -1269,15 +1250,6 @@ def fillna():
         2  0.0  1.0  2.0  3.0
         3  0.0  3.0  2.0  4.0
 
-        Only replace the first NaN element.
```

Comment:
why doesn't this work as well?

Reply:
fillna doesn't support limit

https://github.com/snowflakedb/snowpark-python/blob/75cba1001fd7d09c40ea76a67a0320176cfc4942/src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py#L7856-L7859